templates/Dataland/v2/status-badge.html.twig

```diff
@@ -0,0 +1,20 @@
+<div>
+    {% if dataset.availabilityStatus == constant("App\\Entity\\DatasetSubmission::AVAILABILITY_STATUS_NOT_AVAILABLE") %}
+        <span class="badge badge-secondary">Identified</span>
+    {% elseif dataset.availabilityStatus == constant("App\\Entity\\DatasetSubmission::AVAILABILITY_STATUS_PENDING_METADATA_SUBMISSION")
+        or dataset.availabilityStatus == constant("App\\Entity\\DatasetSubmission::AVAILABILITY_STATUS_PENDING_METADATA_APPROVAL") %}
+        <span class="badge badge-submitted">Submitted</span>
+    {% elseif dataset.availabilityStatus == constant("App\\Entity\\DatasetSubmission::AVAILABILITY_STATUS_RESTRICTED_REMOTELY_HOSTED")
+        or dataset.availabilityStatus == constant("App\\Entity\\DatasetSubmission::AVAILABILITY_STATUS_RESTRICTED") %}
+        <span class="badge badge-restricted">Restricted</span>
+    {% elseif dataset.availabilityStatus == constant("App\\Entity\\DatasetSubmission::AVAILABILITY_STATUS_PUBLICLY_AVAILABLE_REMOTELY_HOSTED")
+        or dataset.availabilityStatus == constant("App\\Entity\\DatasetSubmission::AVAILABILITY_STATUS_PUBLICLY_AVAILABLE") %}
+        <span class="badge badge-available">Available</span>
+    {% endif %}
+    {% if dataset.availabilityStatus == constant("App\\Entity\\DatasetSubmission::AVAILABILITY_STATUS_PUBLICLY_AVAILABLE_REMOTELY_HOSTED") %}
```

Comment:
I think we should also include the badge for remotely hosted if it is AVAILABILITY_STATUS_RESTRICTED_REMOTELY_HOSTED as it is still remotely hosted. (separate logic)